### PR TITLE
feat: add xcm documentation

### PR DESCRIPTION
**Pull request by tomjeatt**
Documentation for transferring KSM from Kusama -> Kintsugi. I've added this to the transfers page, but we can link to the section directly if we need to.

### CODE REVIEW

File: docs/guides/transfers.md [hunk 1]
```diff
@@ -22,13 +22,26 @@ Coming soon
...
 
+### Cross chain transfers
+
+?> At present, we only support transferring KSM from Kusama to Kintsugi
+
+1. Go to the Transfer page on the app. Make sure you allow polkadot.js to connect to the website.
+2. Select the Cross Chain Transfer tab.
+3. Make sure you select the correct account from the account selector at the top of the page. This is the account from which the transfer will be made.
+4. Your available balance on the relay chain is shown above the amount field. This wil be different from the parachain balance shown at the top of the page.
+5. The from and to chain values are automatically set to Kusama and Kintsugi.
```
Comment (alexbleau) [inline docs/guides/transfers.md]: I would maybe reword it to avoid confusion that to Kusama actually means that the "from" field will be Kusama. 

Alternative: The "from" chain value will automatically be set to Kusama and the "to" chain value will be set to Kintsugi

File: docs/guides/transfers.md [hunk 2]
```diff
@@ -22,13 +22,26 @@ Coming soon
...
+6. Select an account from the destination select field. You can only transfer to accounts that are in your wallet.
+7. Click "Transfer" and sign the extrinsic to make a transaction.
+
```
Comment (alexbleau) [inline docs/guides/transfers.md:45]: I would add a last step that explains where they will see the updated KSM:

Suggestion: 8. After the transfer has been made, if you connect to the dapp with the wallet/account that has received the KSM, the top right KSM balance should now be updated to include the amount of KSM transferred.

File: docs/guides/transfers.md (hunk 1 above)
Comment (tomjeatt) [reply docs/guides/transfers.md]: Done :)

**Review by alexbleau — APPROVED** [review]
